src/schemathesis/models.py

```diff
@@ -78,7 +79,7 @@ def formatted_path(self) -> str:
 
     def get_full_base_url(self) -> Optional[str]:
         """Create a full base url, adding "localhost" for WSGI apps."""
-        parts = urlsplit(self.base_url)
+        parts = urlsplit(self.base_url or "")
```

Comment:
It doesn't seem relevant to the topic of this PR. Why is it here?

Reply:
My PRs are failing in mypy due to an optional value being passed to this function

Reply:
Aha, I see. It might be caused by the recent changes in the pre-commit config - I will fix it soon

Reply:
I waiting you fix it?

Reply:
Gimme 5 minutes and then you can rebase your branch

Reply:
Ok, now it should be fine - I reverted the change of the default Python version for pre-commit. Let me know if it works for you